bloom_nofos/nofos/test_nofo.py

```diff
@@ -6683,7 +6683,7 @@ def test_returns_match_with_highlight(self):
         self.assertIn("Deadline Details", match["subsection"].name)
         self.assertEqual(self.section, match["section"])
         self.assertEqual(self.matching_subsection.id, match["subsection"].id)
-        self.assertEqual(self.matching_subsection.body, match["subsection"].body)
+        self.assertEqual(match["subsection"].body, self.matching_subsection.body)
 
```

Comment:
Looks like this line and the next one are causing a merge conflict (if you do `git rebase main`), but easy enough to fix.